### PR TITLE
Fix new forms

### PR DESCRIPTION
**Pull request by arellaTV**
These are changes to the markups provided and tested in staging by APM

Short List: http://scprv4-staging.scprdev.org/short-list/latest
KPCC In Person: https://scprv4-staging.scprdev.org/events/kpcc-in-person

### CODE REVIEW

File: app/views/shared/widgets/_events_subscribe.html.erb [hunk 1]
```diff
@@ -8,11 +8,11 @@
...
         <div class="form-group" style="display: inline-block">
           <input type="email" name="subscription_request[fsm][Form_Email_Address]" id="subscription_request_fsm_Form_Email_Address" required="required" maxlength="254" class="newsletter-email form-control" placeholder="email@somedomain.com" />
-        </div><input name="subscription_request[fsm][Form_BusinessUnit]" type="hidden" value="SCPR" /><input name="subscription_request[fsm][Form_FormName]" type="hidden" value="KPCCInPerson_Newsletter_Sub" /><input name="subscription_request[fsm][Form_Opt_In_Source]" type="hidden" value="SCPR Newsletter" />
-        <input name="subscription_request[publication_lists][489]" type="hidden" value="on" />
+        </div><input name="subscription_request[fsm][Form_BusinessUnit]" type="hidden" value="SCPR" /><input name="subscription_request[fsm][Form_FormName]" type="hidden" value=" KPCCInPerson_Newsletter_Sub" /><input name="subscription_request[fsm][Form_Opt_In_Source]" type="hidden" value="SCPR Newsletter" />
+        <input name="subscription_request[publication_lists][494]" type="hidden" value="on" />
```
Comment (nakedsushi) [inline app/views/shared/widgets/_events_subscribe.html.erb]: What are the numbers here? Are they hard coded in based on something APM gave us?

Comment (arellaTV) [inline app/views/shared/widgets/_events_subscribe.html.erb]: @nakedsushi Yes, I believe they are hardcoded to something APM gave us (possibly a reference id to pass in when the form is submitted to that url).

Comment (arellaTV) [inline app/views/shared/widgets/_events_subscribe.html.erb]: Sort of like the Eloqua Ids from the previous form

**Comment by arellaTV** [comment]
Pulled in and verified to work via APM!

**Pull request closed without merging**